### ISSUE TITLE
Subliminal fails to identify buttons in action sheets in popovers.

### ISSUE THREAD

**Issue by wearhere** [inkling/Subliminal#46]
This is because Subliminal is failing to identify both the popover and the action sheet. Two bugs:
- [x] Subliminal doesn't include action sheets in accessibility paths.
- [x] Subliminal cannot identify popovers while they show action sheets, because Subliminal identifies instances of `_UIPopoverView` by their superview's accessibility label of "dismiss popup" and the superview does not carry that label while an action sheet is showing.

**Comment by wearhere** [comment]
#47 fixes the first bug.

**Comment by wearhere** [comment]
Also fixes the second bug, now.
